mypy.ini

```diff
@@ -1,7 +1,10 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
 [mypy]
 pretty = True
 python_version = 3.8
-mypy_path = $MYPY_CONFIG_FILE_DIR/src:$MYPY_CONFIG_FILE_DIR/lib
+mypy_path = $MYPY_CONFIG_FILE_DIR/src:$MYPY_CONFIG_FILE_DIR/lib:$MYPY_CONFIG_FILE_DIR/tests/unit:$MYPY_CONFIG_FILE_DIR/tests/integration
```

Comment:
Why? You shouldn't be importing from these directories.  Does mypy fail without this?